hack/build.sh

```diff
@@ -2,6 +2,8 @@
 
 set -ex
 
+RHCOS_BUILD_NAME="${RHCOS_BUILD_NAME:-400.7.20190306.0}"
```

Comment:
I don't have a problem with this but note that the version referenced isn't an official cut.

Reply:
> I don't have a problem with this but note that the version referenced isn't an official cut.

I don't care what we pin to, just tell me what you want ;)

Reply:
This is fine. I just wanted to note that it wasn't an officially ID'd build for any specific cut.

Reply:
FWIW `400.7.20190312.0` now exists which includes a `pivot` update among other things. @cgwalters do you need that as part of the pin?

Reply:
My tentative vote here is to ship this as is now and start building "mental muscle memory" or SOP for bumping it immediately after, but it's not a really strong opinion.